client/trino-client/src/test/java/io/trino/client/auth/external/TestExternalAuthentication.java

```diff
@@ -148,26 +143,4 @@ public URI redirectedTo()
             return redirectedTo;
         }
     }
-
-    private static final class MockTokenPoller
```

Comment:
nit: please extract such refactors to separate commits. Now I am not 100% sure what changed in the body as entire class is added to git and other removed.